docarray/typing/tensor/video/abstract_video_tensor.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABC, abstractmethod
+from typing import BinaryIO, TypeVar, Union
+
+import numpy as np
+
+from docarray.typing.tensor.abstract_tensor import AbstractTensor
+
+T = TypeVar('T', bound='AbstractVideoTensor')
+
+
+class AbstractVideoTensor(AbstractTensor, ABC):
+    @abstractmethod
+    def to_numpy(self) -> np.ndarray:
+        """
+        Convert video tensor to numpy.ndarray.
+        """
+        ...
```

Comment:
I'd rather have this in the comp backends, otherwise we might com into conflict with methods that are already provided by torch/tf/...